### ISSUE TITLE
Cannot read property 'numBeats' of undefined

### ISSUE THREAD

**Issue by markknol** [ddj231/Handel#7]
Sorry for amount of issues but I run into this error:
`Cannot read property 'numBeats' of undefined`.
I probably do something wrong but I can't find it.
```
start
	save c = C4, E4, G4 for 2b
	save dm = D4, F4, A4 for 2b
	save em = E4, G4, B4 for 2b
	save f = F4, A4, C5 for 2b
	save g = G4, B4, D5 for 2b
	save am = A4, C5, E5 for 2b
	save bdim = B4, D5, F5 for 2b

	chunk intro
		play g
		play c
	endchunk
	
	chunk chorus
		play g
		play c
		play dm
		play f
	endchunk
	
	run intro with sound piano
	run chorus with sound piano
finish
```

**Comment by ddj231** [comment]
@markknol thanks for this issue as well. It looks like an issue of scoping. Currently chunks are only looking in their own stack frame for variables so when you try to play your global playables the individual chunks can't access those variables. This should be a small fix, by making the stack frames look at their enclosing stack frame (and on) as well for variables. The symbol table already behaves this way which is why you are not getting a `variable does not exist error`.

**Comment by ddj231** [comment]
This is now addressed here 86349677a5266781148d799c720b3dfe52f57cf6, and in Handel (v0.5.6) your above code works as expected.
